src/common/components/profile/UserTransactionCards.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState, useEffect } from "react";
```

Comment:
component file names should start with lowercase always to keep consistency

Reply:
Updated